### PR TITLE
Added is_json() and is_serialized() methods in Str class

### PR DESCRIPTION
**Pull request by billmn**

### CODE REVIEW

**Comment by huglester** [comment]
Was missing these methods :)

+1

**Comment by tomschlick** [comment]
+1 on these, I'm sure it will help get this in faster if you link the docs pull request as well

**Comment by billmn** [comment]
@tomschlick Yeah ... and this is the docs :
https://github.com/fuel/docs/pull/392

File: classes/str.php [hunk 1]
```diff
@@ -355,6 +355,41 @@ public static function tr($string, $array = array())
...
+	/**
+	 * Check if a string is json encoded
+	 * 
+	 * @param  string $string string to check
+	 * @return bool
+	 */
+	public static function is_json($string)
+	{
+		json_decode($string);
+		return json_last_error() == JSON_ERROR_NONE;
```
Comment (frankdejonge) [inline classes/str.php]: == should be ===

**Comment by frankdejonge** [comment]
The rest looks good to me. I thought maybe this would something for the format classes, but it actually makes sense having them in the Str class since that's what we're handling.

File: classes/str.php (hunk 1 above)
Comment (billmn) [inline classes/str.php]: Added ... thanks ;P

**Comment by huglester** [comment]
gogo @FrenkyNet , merge this in!

**Pull request merged**